packages/account/package-lock.json

```diff
@@ -2071,6 +2071,16 @@
       "dev": true,
```

Comment:
Why is this changed here? We don't use `deriv-charts` in account package.

Reply:
@salar-fs interesting, because I also didn't apply anything to account, just run `npm run bootstrap`